vault/blog.2021.2021-08-30-highlights-vscode-159.md

```diff
@@ -0,0 +1,66 @@
+---
+id: JzH76BDLaFwJloUFzkGGg
+title: Highlights from VSCode 1.59
+desc: ''
+updated: 1630272832402
+created: 1630272366498
+date: '2021-08-30'
+image: "https://org-dendron-public-assets.s3.amazonaws.com/images/vscode-release-highlights-159.png"
+excerpt: "Terminal magic tricks, shared view states, and more in the VSCode 1.59 release"
+author: "Derek Ardolf"
+---
+
+> This blog article makes use of [footnotes](https://wiki.dendron.so/notes/ba97866b-889f-4ac6-86e7-bb2d97f6e376.html#footnotes), which can be useful to [academics using Dendron](https://wiki.dendron.so/notes/77c8f69c-32b8-4331-b1b2-c4211dcb9fb7.html) or for a cleaner blog experience.
+
+The open source VSCode project, along with VSCodium[^1], completed their latest monthly release with v1.59[^2]. Let's take a look at some features that the insiders build[^3] had their impatient, bleeding-edge hands on before the August 2021 release is announced.
+
+### Share view state
+
+VSCode users are often bouncing around different files in their workspaces. By default, if you close out a file tab, VSCode doesn't track where you were in that file. This can be annoying if you didn't mean to close out the file.
+
+Say hello to shared view states across editor groups[^4]. Editor groups are the windows of tab collections within the VSCode editor[^5] that grow whenever **Split Editor/Open to the Side** commands are used. These groups can remember your past so that your rampant mind doesn't need to.
+
+![GIF showing the share view state feature with The House on The Borderland, by William Hope Hodgson](https://org-dendron-public-assets.s3.amazonaws.com/images/vscode-share-view-state.gif)
+
+> Ah yes, that's right, I left off at the pig people.[^6]
+
+### Terminal magic tricks
+
+By default, a new terminal can be created with `Ctrl+Tilde(~)`, where it pops up in the bottom as an entryway into the powerful, dark underbelly of shells. VSCode 1.59 highlights some wizardry alternatives.
+
+#### Create terminals to the side
+
+Are you one of those people who spins a monitor into portrait mode, because of some insatiable love for 90<sup>o</sup> angles? Maybe you simply prefer tall terminals? `Ctrl+Shift+P` and `Terminal: Create Terminal in Editor Area to the Side`[^7], will be exactly what you're looking for.
+
+![GIF showing the creation of terminals to the side, instead of defaulting to the bottom](https://org-dendron-public-assets.s3.amazonaws.com/images/vscode-side-terminals.gif)
+
+#### Drag and drop terminals across windows
+
+How many VSCode windows are sitting open on your desktop? Are you taking notes in one workspace, while continuing your commands in a separate VSCode window? Make your life easier, retain all your in-shell variables, and drag the active terminal from one window to another.
```

Comment:
remember when typos where just typos where the worst thing that could happen is other people making fun off your lack of oxford comma's? with terminal editors, your typo can now shutdown your production database :)

Reply:
Snark upgrade applied